### PR TITLE
Use headings semantically

### PR DESCRIPTION
**Pull request by frederike-ramin**
# Short Description
- We previously didn't use headings in the correct increasing order. Now we do^^

# Changes
- Went through all templates and adapted the headings so that they are consecutive.
- Added bootstrap's `.h<x>` classes where necessary to contain the look and feel.
- Added a heading_level parameter to accordion and teaser card to be able to configure the level differently for each page.

# Feedback
- How cool is the `h{{ heading_level }}` part?! I thought I'd just try it and it works! So nice. :D

### CODE REVIEW

File: webapp/app/templates/content/data_privacy.html [hunk 1]
```diff
@@ -54,31 +54,31 @@ <h2 class="pb-1">{{ _('data_privacy.rights-title') }}</h2>
         <p>{{ _('data_privacy.rights-text') }}</p>
 
         <div class="mt-5">
-            <h4>{{ _('data_privacy.rights-information') }}</h4>
+            <h3 class="h4">{{ _('data_privacy.rights-information') }}</h3>
```
Comment (JulianRoesner) [inline webapp/app/templates/content/data_privacy.html:57]: Although the use of the h4 class is clever, I am not such a big fand to be honest. I would say that we probably should control the look through our own class and use our variables there. We might not have to do this now but it seems cleaner to me.

WDYT?

File: webapp/app/templates/content/contact.html [hunk 2]
```diff
@@ -6,19 +6,19 @@ <h1 class="my-4">{{ _('contact.heading') }}</h1>
...
 
     <div class="mt-5 mb-5">
-        <h3>{{ _('contact.feedback-headline') }}</h3>
+        <h2>{{ _('contact.feedback-headline') }}</h2>
         <p class="mt-2">{{ _('contact.feedback-text') }}</p>
     </div>
 
     <div class="mt-5 mb-5">
-        <h3>{{ _('contact.security-headline') }}</h3>
+        <h2>{{ _('contact.security-headline') }}</h2>
```
Comment (JulianRoesner) [inline webapp/app/templates/content/contact.html:21]: This does also change the look. So we might have to either also use the h3 class here or think of another approach?

**Review by JulianRoesner — COMMENTED** [review]
Very nice idea with the heading_level, I like it a lot. We might have to think/talk a bit about how we want to do the styling now (s. comments)

File: webapp/app/templates/content/data_privacy.html (hunk 1 above)
Comment (frederike-ramin) [reply webapp/app/templates/content/data_privacy.html:57]: I mean, we can just override the bootstrap classes if we want them to look another way?

Comment (frederike-ramin) [reply webapp/app/templates/content/data_privacy.html:57]: We talked about it and I understood the implications of it. However, we decided to leave it at that for now and get to it later at a larger scale when splitting up the css files / using sass instead. I added a comment to the corresponding ticket.

**Pull request merged**